Description: Fix keysize of aes128-gcm and aes192-gcm
 This is
 https://github.com/robrichards/xmlseclibs/pull/213
 https://github.com/robrichards/xmlseclibs/pull/213/commits/a50d550805388cbdfc4881a2fc91ef09c32f100a
 See further
 https://github.com/simplesamlphp/saml2/issues/179#issuecomment-687597903
--- a/vendor/robrichards/xmlseclibs/src/XMLSecurityKey.php	2020-04-22 19:19:51.000000000 +0200
+++ b/vendor/robrichards/xmlseclibs/src/XMLSecurityKey.php	2021-01-18 12:19:25.846384184 +0100
@@ -151,7 +151,7 @@
                 $this->cryptParams['cipher'] = 'aes-128-gcm';
                 $this->cryptParams['type'] = 'symmetric';
                 $this->cryptParams['method'] = 'http://www.w3.org/2009/xmlenc11#aes128-gcm';
-                $this->cryptParams['keysize'] = 32;
+                $this->cryptParams['keysize'] = 16;
                 $this->cryptParams['blocksize'] = 16;
                 break;
             case (self::AES192_GCM):
@@ -159,7 +160,7 @@
                 $this->cryptParams['cipher'] = 'aes-192-gcm';
                 $this->cryptParams['type'] = 'symmetric';
                 $this->cryptParams['method'] = 'http://www.w3.org/2009/xmlenc11#aes192-gcm';
-                $this->cryptParams['keysize'] = 32;
+                $this->cryptParams['keysize'] = 24;
                 $this->cryptParams['blocksize'] = 16;
                 break;
             case (self::AES256_GCM):
